go/consensus/api/transaction/testvectors/testvectors.go

```diff
@@ -11,27 +11,33 @@ import (
 
 const keySeedPrefix = "oasis-core test vectors: "
 
-// TestVector is a staking message test vector.
-type TestVector struct {
-	Kind             string                        `json:"kind"`
-	SignatureContext string                        `json:"signature_context"`
-	Tx               interface{}                   `json:"tx"`
-	SignedTx         transaction.SignedTransaction `json:"signed_tx"`
-	EncodedTx        []byte                        `json:"encoded_tx"`
-	EncodedSignedTx  []byte                        `json:"encoded_signed_tx"`
-	Valid            bool                          `json:"valid"`
-	SignerPrivateKey []byte                        `json:"signer_private_key"`
-	SignerPublicKey  signature.PublicKey           `json:"signer_public_key"`
+// TestVector is an Oasis transaction test vector.
+type TestVector struct { // nolint: maligned
+	Kind             string      `json:"kind"`
+	SignatureContext string      `json:"signature_context"`
+	Tx               interface{} `json:"tx"`
+	// ValidTx indicates whether the transaction is (statically) valid.
+	// NOTE: This means that the transaction passes basic static validation, but
+	// it may still not be valid on the given network due to invalid nonce,
+	// or due to some specific parameters set on the network.
+	ValidTx         bool                          `json:"valid_tx"`
```

Comment:
Don't forget to update docs on test vectors (`docs/consensus/test-vectors.md`).

Reply:
Thanks! Done.